fix: clarification, "using the OpenShift web console" (#2326)
codenvy/che-docs: modules/administration-guide/pages/upgrading-che-using-the-web-console.adoc

:_content-type: ASSEMBLY
:navtitle: Upgrading {prod-short} using the web console
:keywords: administration guide, upgrading-che-using-operatorhub
:page-aliases:

[id="upgrading-{prod-id-short}-using-the web console_{context}"]
= Upgrading {prod-short} using the web console

> This section describes how to mannually approve an upgrade from an earlier minor version using the {prod} Operator from the Red Hat Ecosystem Catalog in the OpenShift web console.

Author: ```suggestion
You can manually approve an upgrade from an earlier minor version using the {prod} Operator from the Red Hat Ecosystem Catalog in the OpenShift web console.
```
